workers/data_refinery_workers/processors/smasher_email.html

```diff
@@ -0,0 +1,1501 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Title</title>
+    <style>
```

Comment:
I took a closer look at the style sheet in this file. You may want to correct the indentations, and some of the lines are redundant. For example, `<h6>` is not being used, but the style has some customization. Maybe we should ask @dvenprasad or @arielsvn to simplify this file?

Reply:
@arielsvn you can make your changes directly to this branch, just be mindful of the `REPLACEME` links